### DISCUSSION TITLE
Neper 4.1.2 / Neper 4.0.2

### DISCUSSION THREAD

**Discussion by Sofia-Y** [neperfepx/neper#117] [Q&A] [answered]
Hello, @rquey !

I installed the new version of Neper 4.1.2, however, when I run my examples they still use the previous version Neper 4.0.2. How is this possible and what could I possibly have done wrong?

Thanks a lot for your assistance!

**Comment by rquey** [comment, marked as answer]
This is an installation/system issue, not a Neper issue.

You are simply not using the new binary you compiled (v4.1.2). Did you run `make install`?

I assume you are calling Neper as `neper` on the command line. Run `which neper` to see what file this corresponds to.  This file is your  Neper v4.0.2. It is to be removed or replaced by the new one (v4.1.2).

**Reply by Sofia-Y** [reply]
Thank you for your reply. I am working on it.
